pycoast/tests/test_pycoast.py

```diff
@@ -640,6 +641,7 @@ def test_add_polygon_agg(self):
                        outline='green', fill='gray', width=2)
         cw.add_coastlines(img, area_def, resolution='l', level=4)
 
+        img.save(os.path.join(os.path.dirname(__file__), 'nh_polygons_agg_test.png'))
```

Comment:
You are just replacing the test reference, right ?

Reply:
Crap, I did so many commits without accidentally doing this. Ugh. I'll remove it.